files/server.xml.j2

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
```

Comment:
@fabricetriboix I think we don't require parameterizing the server.xml config file. Since tomcat takes care of replacing the environment variables/system variables. Could you please try without parameterization with jinja2?

Reply:
I forgot about that. Anyway, I am still not clear on what is possible and what is not possible on that front. In addition, I would like to use jinja2 for this Docker image so we can use it as a reference going forward. If that is not suitable, I can investigate getting Tomcat to expand environment variables, but I would like to keep jinja2 for this PR.

Reply:
Okay. Just opening an issue here in the repo to keep a reminder. Is that okay.?

Reply:
@naveen-armedia I don't understand at all. You initially said that you are OK with me using jinja2 in order to have a reference implementation and I can change it later if I manage to get Tomcat to expand environment variables. Now you are saying the exact opposite. Or am I misinterpreting something?

Reply:
I would argue that if we can leverage environment variables or similar dynamic substitutions on the final files without needing a template engine to preprocess them, then this should be the preferred path. That said, definitely any such transition can be delayed if appropriate to facilitate progress.

Reply:
@fabricetriboix which comments of mine are contradicting you. I totally agree with the environment variable substitution with jinja2 where it is appropriate. But If the tomcat or other application can handle environment variable substitution we should avoid using jinja2.

Reply:
@drivera-armedia our arkcase stack(snowbound, arkcase core, Pentaho, cloud-config) were tomcat-based applications. But application like Pentaho runs on old tomcat versions like 8.5.27 where environment variables or system variables substitution is not possible in most of the config files. The only place where we can add variables is to add in `Catalina.properties` file( there might be other ways too). So in these old applications, we could leverage environment variable substitution with jinja2. Saying that in the last meeting `Discuss parametrization of docker images` led by Fabrice, we agreed to take reference of using jinja2 implementation with snowbound dockerimage.

Reply:
Right. This is consistent with what I mean and I agree with this approach. I guess what I'm saying is that jinja2 should be considered the backup plan in the event that we can't make environment expansion work OOTB in whatever it is we're deploying ... does that make more sense?